ptf_tests/common/utils/test_utils.py

```diff
@@ -1021,3 +1041,123 @@ def host_check_netperf(remote=False, hostname="",username="",password=""):
     
     connection.tear_down()
     return True
+
+
+def send_ctrl_c(conn):
+
+    conn.sendCmd('\x03')
+
+    return True
+
+def send_scapy_traffic_from_vm(vm_id,conn,remote_conn,config_data,traffic_type="unicast"):
+
+    vm = config_data['vm'][vm_id]
+    port = config_data['port'][vm_id]
+    
+    num = config_data['traffic']['number_pkts'][0]
+    eth_src = '\"%s\"' % port['mac'] 
+    eth_dst = '\"%s\"' % vm['mac_remote']
+    ip_src = '\"%s\"' % vm['dst_gw']
+    ip_dst= '\"%s\"' % vm['remote_ip']
+    iface = '%s' % port['interface']
+    pcap_file= config_data['traffic']['pcap_file_name'][0]
+
+    if traffic_type=='multicast':
+        ip_dst= '\"%s\"' % config_data['traffic']['in_pkt_header']['ip_dst'][1]
+        eth_dst='\"01:00:5e:00:00:01\"'
+    elif traffic_type=='broadcast':
+        ip_dst= '\"%s\"' % config_data['traffic']['in_pkt_header']['ip_dst'][2]
+        eth_dst='\"FF:FF:FF:FF:FF\"'
+    
+    #packet build    
+    command_list = ['python3','import scapy.all as scapy']
+    
+    for cmd in command_list:
+        status = conn.sendCmd(cmd)
+        if status:
+            print(f"Command: {cmd} executed ")
+        else:
+            print("Failed to execute command {cmd}")
+    
+    #starting log on remote vmcmd="open(\"%s\").read()" %pcap_file
+    print(f"capturing in to pcap file {pcap_file} on remote_host")
+    tcpdump_cmd = "tcpdump -i %s -w %s" % (iface, pcap_file)
+    #remote_conn.sendCmd('tcpdump -i ens3 -w /tmp/log_cap1.pcap &')
+    remote_conn.sendCmd(tcpdump_cmd)
+
+    # Packet Building using scapy
+    conn.sendCmd('pkt=scapy.Ether(dst=%s,src=%s)/scapy.IP(dst=%s,src=%s)/\
+                  scapy.TCP()/scapy.Raw(\'0\'*50)' %(eth_dst,eth_src,ip_dst,ip_src))
+
+    print('scapy.sendp(pkt,iface=\"%s\",count=%s)' % (iface,num))
```

Comment:
You can make this more readable. E.G. Sending scapy traffic from iface {iface} packet count {count}

Reply:
Modified this statement  print('Sending scapy traffic on iface=\"%s\",count=%s' % (iface,num))